.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,3 +7,12 @@
 
 
 ### Specify the version of the product this PR applies to. 
+
+
+### Pull Request Checklist
+
+Before merging the PR the following items must be checked:
+
+- [ ] `vale` has been run successufully against the PR branch
```

Comment:
What does running vale against the branch mean here? If I run vale against the current master, I see
```
635 errors, 896 warnings and 1153 suggestions in 548 files.
```
Perhaps something like 
```
`vale` has been run successfully against files modified in the PR
```
is clearer.

On the other hand, this requirement will mean having to rewrite docs pages if we e.g. just move things around (extract a section to a procedure, or move something from Overview to EUG).

Reply:
Better to say: run vale againts the changed files ...